VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -57,6 +57,7 @@ def __init__(self,patching, hutil, logger,para_parser):
         self.isAnyDiskExcluded = False
         self.LunListEmpty = False
         self.logicalVolume_to_bill = []
+        self.sudo_off = 0 # run the commands using sudo
```

Comment:
Suggestion:

Since we are including this config - we can do a better default initialization and check before even allowing toggle of this config. Checking if sudo is installed can be done with:
```bash
command -v sudo
```
The above will output an empty string if `sudo` is not found in PATH - which almost always means that it isn't installed.
One problem that could arise though is if the command `command` itself is not available - I have never faced this in practice however. LISA should be able to help us verify this if we decide to go ahead with this.

Reply:
since our extension runs with sudo access, checking for the existence of sudo within the extension itself may not be necessary. 
However, in LISA it might be a good idea to include the check for sudo existence.